### PR TITLE
[DownloadBuildArtifactsv0] Fixed Missing log : Downloading artifact ArtifactName from: ArtifactPath

### PR DESCRIPTION
**Pull request by rahulroy9202**
add missing log 

https://github.com/microsoft/azure-pipelines-tasks/issues/14704

**Task name**: DownloadBuildArtifacts

**Description**: Fixed missing log

**Documentation changes required:** No

**Added unit tests:**  No

**Attached related issue:** Yes #14704

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DownloadBuildArtifactsV0/task.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 186,
+        "Minor": 187,
```
Comment (anatolybolshakov) [inline Tasks/DownloadBuildArtifactsV0/task.json:12]: Could you please bump it as 0.186.2 instead, and for task.loc.json file also?

**Review by anatolybolshakov — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
Thanks for contribution! Could you please take a look at comments?

File: Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerFilePath.ts [hunk 2]
```diff
@@ -23,7 +23,9 @@ export class DownloadHandlerFilePath extends DownloadHandler {
         const downloadUrl = this.config.artifactInfo.resource.data;
         const artifactName = this.config.artifactInfo.name.replace('/', '\\');
         let artifactLocation = path.join(downloadUrl, artifactName);
-
+        
+        console.log(tl.loc('DownloadArtifacts', artifactName, artifactLocation));
+
```
Comment (DaniilShmelev) [inline Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerFilePath.ts:28]:
```suggestion

        console.log(tl.loc('DownloadArtifacts', artifactName, artifactLocation));

```

**Review by DaniilShmelev — APPROVED** [review]
Take a look at the comments, but other than that LGTM

**Comment by rahulroy9202** [comment]
> 
> 
> Thanks for contribution! Could you please take a look at comments?

I am on it.

File: Tasks/DownloadBuildArtifactsV0/task.json (hunk 1 above)
Comment (rahulroy9202) [reply Tasks/DownloadBuildArtifactsV0/task.json]: Done!

File: Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerFilePath.ts (hunk 2 above)
Comment (rahulroy9202) [reply Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerFilePath.ts]: Cleaned up the whitespaces. Thank You

**Comment by rahulroy9202** [comment]
All feedback incorporated. Thank You

File: Tasks/DownloadBuildArtifactsV0/task.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 186,
+        "Minor": 186.2,
         "Patch": 1
```
Comment (DaniilShmelev) [inline Tasks/DownloadBuildArtifactsV0/task.json:13]:
```suggestion
        "Minor": 186,
        "Patch": 2
```

File: Tasks/DownloadBuildArtifactsV0/task.loc.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 186,
+    "Minor": 186.2,
     "Patch": 1
```
Comment (DaniilShmelev) [inline Tasks/DownloadBuildArtifactsV0/task.loc.json:13]:
```suggestion
    "Minor": 186,
    "Patch": 2
```

Comment (rahulroy9202) [reply Tasks/DownloadBuildArtifactsV0/task.loc.json]: done for both files.

Comment (rahulroy9202) [reply Tasks/DownloadBuildArtifactsV0/task.loc.json]: Thank You

**Review by DaniilShmelev — APPROVED** [review]

**Comment by rahulroy9202** [comment]
> 
> 
> /azp run

Need to rerun?

image

**Comment by rahulroy9202** [comment]
Please rerun the pipeline!

`It failed due to an infrastructure failure. Notification of assignment to an agent was never received.`

**Review by EzzhevNikita — APPROVED** [review]

**Pull request merged**